src/odemis/driver/tmcm.py

```diff
@@ -368,6 +376,23 @@ def __init__(self, name, role, port, axes, ustepsize, address=None,
             except HwError as ex:
                 # Probably some old error left-over, no need to worry too much
                 logging.warning(str(ex))
+
+        # Add digital output axes
+        self._do_axes = do_axes or {}
+        self._led_prot_do = led_prot_do or {}
+        for port, vals in self._do_axes.items():
+            ax_name, high_pos, low_pos, dur = vals
+            axes_def[ax_name] = model.Axis(choices={low_pos, high_pos})
+            self._name_to_do_axis[ax_name] = port
+
+        self._leds_on = model.BooleanVA(False)  # State of refswitch
```

Comment:
There doesn't seem any need for it to be a VA. A simple boolean should be sufficient.